com.haxxed.BasiliskII.yml

```diff
@@ -0,0 +1,44 @@
+app-id: com.haxxed.BasiliskII
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: BasiliskII
+finish-args:
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=home
```

Comment:
Does it require full home access or can it be limited to specific dirs or at least read-only?

Reply:
It read/writes VM disk images and reads ROM files that could potentially be anywhere. It also currently uses old school ~/.basilisk* files for preferences and state. For this to be a drop-in replacement for an existing Basilisk II user, home access would seem to be minimum. The user selects files once in the GUI then they are remembered so I don't know how portals work out with that use case. Also does the portal stuff work with GTK2? Porting to GTK3 is possible but that's a whole lot of work. I was hoping to get Basilisk II back in the hands of users, which would justify spending further time rewriting large portions of it. :(

As it is I keep my vintage Mac stuff on shares mounted in /srv or /run/user/*/gvfs so i'm already having to set overrides for my use case.

I'm open to suggestions how to handle this better. :)

Reply:
Yeah, I was just double-checking I'm not missing anything.